### PR TITLE
Fix links for k/design-proposals-archive

### LINKED ISSUE #35037: Issue with k8s.io/zh-cn/docs/tasks/administer-cluster/namespaces/
https://kubernetes.io/zh-cn/docs/tasks/administer-cluster/namespaces/
[名字空间的设计文档](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/namespaces.md)
404

### PR DESCRIPTION
**Pull request by fusida**
The contributors/design-proposals in k/community was removed. It's only
available in k/design-proposals-archive repo now.

reforence #34155
fix #35037

### CODE REVIEW

**Comment by fusida** [comment]
cc @Sea-n @jihoon-seo

File: content/zh-cn/docs/reference/using-api/_index.md [hunk 1]
```diff
@@ -59,11 +59,11 @@ JSON 和 Protobuf 序列化模式遵循相同的模式更改原则。
 
 <!--
 The API versioning and software versioning are indirectly related.
-The [API and release versioning proposal](https://git.k8s.io/community/contributors/design-proposals/release/versioning.md)
+The [API and release versioning proposal](https://git.k8s.io/design-proposals-archive/release/versioning.md)
 describes the relationship between API versioning and software versioning.
 -->
 API 版本控制和软件版本控制是间接相关的。
-[API 和发布版本控制提案](https://git.k8s.io/community/contributors/design-proposals/release/versioning.md)
+[API 和发布版本控制提案](https://git.k8s.io/design-proposals-archive/release/versioning.md)
```
Comment (jihoon-seo) [inline content/zh-cn/docs/reference/using-api/_index.md:66]: Please see #34976:
> [release/versioning.md](https://git.k8s.io/design-proposals-archive/release/versioning.md#kubernetes-release-versioning) that was in the k/design-proposals-archive repository in the past has been migrated to [k/sig-release repository](https://github.com/kubernetes/sig-release/blob/master/release-engineering/versioning.md?rgh-link-date=2022-07-13T06%3A58%3A09Z) repository.

And the suggestion for this file will be applied by #34996.

File: content/zh-cn/releases/release.md [hunk 2]
```diff
@@ -219,7 +219,7 @@ The general labeling process should be consistent across artifact types.
   referring to a release MAJOR.MINOR `vX.Y` version.
 
   See also
-  [release versioning](/contributors/design-proposals/release/versioning.md).
+  [release versioning](https://git.k8s.io/design-proposals-archive/release/versioning.md).
```
Comment (jihoon-seo) [inline content/zh-cn/releases/release.md:222]: Same here.

File: content/zh-cn/releases/release.md [hunk 3]
```diff
@@ -233,7 +233,7 @@ The general labeling process should be consistent across artifact types.
   [GitHub 里程碑](https://help.github.com/en/github/managing-your-work-on-github/associating-milestones-with-issues-and-pull-requests)
   指的是发布 主.次 `vX.Y` 版本。
 
-  另请参阅[发布版本控制](/contributors/design-proposals/release/versioning.md)。
+  另请参阅[发布版本控制](https://git.k8s.io/design-proposals-archive/release/versioning.md)。
```
Comment (jihoon-seo) [inline content/zh-cn/releases/release.md:236]: Same here.

File: content/zh-cn/releases/version-skew-policy.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ Specific cluster deployment tools may place additional restrictions on version s
 ## Supported versions
 
 Kubernetes versions are expressed as **x.y.z**, where **x** is the major version, **y** is the minor version, and **z** is the patch version, following [Semantic Versioning](https://semver.org/) terminology.
-For more information, see [Kubernetes Release Versioning](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#kubernetes-release-versioning).
+For more information, see [Kubernetes Release Versioning](https://git.k8s.io/design-proposals-archive/release/versioning.md#kubernetes-release-versioning).
```
Comment (jihoon-seo) [inline content/zh-cn/releases/version-skew-policy.md:35]: Same here.

File: content/zh-cn/releases/version-skew-policy.md [hunk 5]
```diff
@@ -41,7 +41,7 @@ The Kubernetes project maintains release branches for the most recent three mino
 Kubernetes 版本以 **x.y.z** 表示，其中 **x** 是主要版本，
 **y** 是次要版本，**z** 是补丁版本，遵循[语义版本控制](https://semver.org/)术语。
 更多信息请参见
-[Kubernetes 版本发布控制](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#kubernetes-release-versioning)。
+[Kubernetes 版本发布控制](https://git.k8s.io/design-proposals-archive/release/versioning.md#kubernetes-release-versioning)。
```
Comment (jihoon-seo) [inline content/zh-cn/releases/version-skew-policy.md:44]: Same here.

File: content/zh-cn/docs/reference/using-api/_index.md [hunk 6]
```diff
@@ -59,11 +59,11 @@ JSON 和 Protobuf 序列化模式遵循相同的模式更改原则。
 
 <!--
 The API versioning and software versioning are indirectly related.
-The [API and release versioning proposal](https://git.k8s.io/community/contributors/design-proposals/release/versioning.md)
+The [API and release versioning proposal](https://git.k8s.io/design-proposals-archive/release/versioning.md)
```
Comment (jihoon-seo) [inline content/zh-cn/docs/reference/using-api/_index.md:62]: Same here.

**Review by jihoon-seo — COMMENTED** [review]
@fusida PTAL!

File: content/zh-cn/docs/reference/using-api/_index.md (hunk 1 above)
Comment (fusida) [reply content/zh-cn/docs/reference/using-api/_index.md]: thanks!

**Comment by fusida** [comment]
@jihoon-seo thanks
cc @ayatk

**Comment by tengqm** [comment]
This PR touches 26 files. Are we sure that all those files are currently in sync with the English version?

**Comment by fusida** [comment]
> This PR touches 26 files. Are we sure that all those files are currently in sync with the English version?

reforence https://github.com/kubernetes/website/pull/34155, only resync links not contents

**Comment by Sea-n** [comment]
This PR now contains commits from other PRs, please do `git rebase` and pick only your own commits.

**Comment by fusida** [comment]
remove /language en

**Comment by fusida** [comment]
@Sea-n thanks

**Comment by tengqm** [comment]
/close

I'm closing this PR because we don't accept "partial fixes" to pages. This PR is touching over 20 pages without fully checking whether those pages are still in sync with English upstream. We'd live with these bad links and fix them on a page-by-page basis.

Thanks for the PR anyway.

**Pull request closed without merging**